package.json

```diff
@@ -25,9 +28,10 @@
   "license": "ISC",
   "dependencies": {
     "babel-polyfill": "^6.23.0",
+    "babel-runtime": "^6.23.0",
```

Comment:
the docs are [here](https://babeljs.io/docs/plugins/transform-runtime/). basically what it does is ensure that there is less redundancy in the 'helper' functions that babel uses when transpiling code (i.e. some things can't be transformed to plain ES5 code without some sort of helper function, such as generators and the like) as well as polyfills for things. It also ensures that all polyfills and such do not pollute the global namespace, so if someone wants to use this library and they are using a different polyfill for, say, `Map`, than the one included with `babel-polyfill`, our code will not pollute the global namespace and should be able to exist alongside their existing, polyfilled code.